package.json

```
@@ -33,6 +33,9 @@
     "test:lint": "standard",
     "test": "standard && ava"
   },
+  "peerDependencies": {
+    "keyv": "^4.0.0"
```

We could specify that both versions are supported like this:
```suggestion
    "keyv": "^3.0.0 || ^4.0.0"
```